media-libs/opensubdiv/opensubdiv-3.1.0.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit cmake-utils toolchain-funcs versionator
+
+DESCRIPTION="An Open-Source subdivision surface library"
+HOMEPAGE="http://graphics.pixar.com/opensubdiv/"
+
+MY_PV="$(replace_all_version_separators '_')"
+
+SRC_URI="https://github.com/PixarAnimationStudios/OpenSubdiv/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="ZLIB"
+SLOT="0"
+IUSE="cuda doc examples opencl openmp ptex tbb test tutorials"
+
+# OpenCL does not work with Open Source drivers.
+RDEPEND="media-libs/glew:=
+	media-libs/glfw:=
+	opencl? ( x11-drivers/ati-drivers:* )
+	cuda? ( dev-util/nvidia-cuda-toolkit:* )
+	ptex? ( media-libs/ptex )"
+
+DEPEND="${RDEPEND}
+	tbb? ( dev-cpp/tbb )
+	doc? ( dev-python/docutils app-doc/doxygen )"
+
+KEYWORDS="~amd64 ~x86"
+
+S=${WORKDIR}/OpenSubdiv-${MY_PV}
+
+PATCHES=( "${FILESDIR}"/${P}-skip-osd-regression.patch )
+
+pkg_pretend() {
+	use openmp && tc-check-openmp
```

Comment:
I added the check to pkg_setup, but isn't the default type for MERGE_TYPE source? This is a source build.